### DISCUSSION TITLE
Hope to add support for Unisat Wallet.

### DISCUSSION THREAD

**Discussion by qinyuhua** [rainbow-me/rainbowkit#1892] [Ideas]
Hope to add support for Unisat Wallet.

**Comment by 0xmkh** [comment]
@qinyuhua Does Unisat Wallet support EVM chains ?

**Reply by 0xmkh** [reply]
@qinyuhua Any updates ?

**Reply by nnmax** [reply]
Sorry, UniSat Wallet only supports BTC chains.

**Reply by 0xmkh** [reply]
Got it. We'd only prefer having EVM chains support for wallets, but lmk if you have any other wallets you're interested in 🙏
